src/mapeo-project.js

```diff
@@ -501,6 +504,104 @@ export class MapeoProject {
   get $icons() {
     return this.#iconApi
   }
+
+  /** @param {Object} opts
+   *  @param {string} opts.configPath
+   */
+  async importConfig({ configPath }) {
+    try {
+      await fs.stat(path.resolve(configPath))
+    } catch (e) {
+      console.log(`error loading config file ${configPath}`, e)
+    }
+    const zip = await yauzl.open(configPath)
+    // check for already present presets and delete them if exists
+    // TODO: do the same for fields and icons?
```

Comment:
should I also delete every field and icon? or does it makes more sense to get the `{iconName, iconId}` and `{fieldName, fieldId}` so that when loading presets we directly reference those?

Reply:
For now I implemented the first case (basically delete every preset, field and doc). Matching icons of fields against names doesn't seem to be super reliable...

Reply:
Yes I would delete every field. For icons it would be nice to re-use them somehow (e.g. checking hashes) but for now let's just delete them all.